### PR TITLE
Deprecation warnings

### PR DESCRIPTION
**Pull request by emord**
Gets rid of some deprecation warnings for django 1.7

### CODE REVIEW

File: corehq/ex-submodules/couchforms/models.py [hunk 1]
```diff
@@ -464,3 +467,7 @@ class UnfinishedSubmissionStub(models.Model):
     timestamp = models.DateTimeField()
     saved = models.BooleanField(default=False)
     domain = models.CharField(max_length=256)
+
+    class Meta:
+        app_label = 'couchforms'
+
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/models.py]: W391 blank line at end of file

**Comment by dannyroberts** [comment]
Thanks for pumping this out so quick, @emord. I just fixed a typo, feel free to self merge once tests pass

**Comment by czue** [comment]
i think the build fail is real
```
======================================================================
FAIL: test_delete_doc (mvp_docs.tests.test_pillows.IndicatorPillowTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "custom/_legacy/mvp_docs/tests/test_pillows.py", line 115, in test_delete_doc
    form.save()
  File "custom/_legacy/mvp_docs/models.py", line 59, in save
    assert self.get_db().uri != XFormInstance.get_db().uri
AssertionError
----------------------------------------------------------------------
```

**Comment by dannyroberts** [comment]
I forget, is this waiting on something still?

**Comment by emord** [comment]
I wanted to double check all the models that have non default couchdb databases.

We figure out hte database to write to based on app_label so when I explicitly input XFormInstance app_label mvp_docs was about to write to the wrong DB (yay tests). I already checked them over the weekend, but it was a little rushed and was going to check them again. Will ping when I do that (unless someone else wants to do it)

**Comment by dannyroberts** [comment]
Right. Ok, thanks for the explanation

File: custom/m4change/models.py [hunk 2]
```diff
@@ -52,6 +52,9 @@ class BaseM4ChangeCaseFluff(fluff.IndicatorDocument):
     document_filter = ORFilter(_get_all_m4change_forms())
     domains = M4CHANGE_DOMAINS
     save_direct_to_sql = True
+
```
Comment (dimagimon) [inline custom/m4change/models.py]: W293 blank line contains whitespace

**Comment by emord** [comment]
Ok. this should be ok to go.

One question though: https://github.com/dimagi/commcare-hq/blob/cfbcac2455c08d4e214037dfe5423d31499bbc71/settings.py#L1115-L1118

There are two different couchdb_apps entries for fluff. I think that's an error and should be able to be removed since i explicitly delcared app labels in opm and they were already declared in bihar. Hesitant to remove it though

**Comment by dannyroberts** [comment]
@emord those two are supposed to be there, for the following reason: each project that uses fluff has its own cloudant db. That db needs to have both that domain's models/views, as well as fluffs *views*. I'm not totally sure why bihar and opm do need them while some of the other domains that use fluff don't. So while my rationale may no longer apply, I do know that it wasn't at least initially a mistake.

**Pull request merged**